pegen/parse_string.c

```diff
@@ -445,7 +457,9 @@ static void fstring_shift_children_locations(expr_ty n, int lineno, int col_offs
 static void fstring_shift_expr_locations(expr_ty n, int lineno, int col_offset)
 {
     n->col_offset = n->col_offset + col_offset;
-    n->end_col_offset = n->end_col_offset + col_offset;
+    if (n->lineno == n->end_lineno) {
```

Comment:
No, this is not the only case, where this is true. For example, in the fstring

```
f'''
{
a
}
'''
```

the condition `n->lineno == n->end_lineno` holds, while in the fstring

```
f'''
{
f(
    a, b, c
)
}
'''
```

it is not true.

Reply:
I think that it would help to add a comment explaining roughtly what situations are we correcting here because is not inmediately obvious, but this is a "nice to have" so if you find it not straightfoward to describe is fine :)